### ISSUE TITLE
Add controls for "go forwards slightly" and "go backwards slightly"

### ISSUE THREAD

**Issue by eyalroz** [TeamNewPipe/NewPipe#7714]
### Checklist

- [x] I checked, but didn't find any duplicates (open OR closed) of this issue in the repo. 
- [x] I have read the contribution guidelines given at https://github.com/TeamNewPipe/NewPipe/blob/HEAD/.github/CONTRIBUTING.md.
- [x] This issue contains only one feature request. I will open one issue for every feature I want to request.

#### What feature do you want?
Controls for going forward and backwards by a few seconds (10, 15, 30 something like that) in a video.

Below is a screenshot of the playing UI of the PodVerse app, which has such controls. I don't suggest adopting that as the UI of NewPipe, this is an example for some inspiration of having such controls.

![2](https://user-images.githubusercontent.com/8672255/151667071-8315c676-6d9e-43a6-a132-c90ec4e8ad7b.png)

#### Why do you want this feature?

Two reasons:

1. When playing a video, you often miss some short bit of it, being distracted for a few seconds or hearing some external noise, and you want to go back slightly, while continuing play.  Doing this by using the whole-video navigation control is cumbersome, requiring you to keep your finger pressed, pay close attention to the reported time index, and not make a wrong movement when releasing. It would be much more convenient to press a simple button/control for "skip a few seconds back".
2.   When playing a video, you often want to skip ahead slightly, to see if maybe the boring part is over. As in the previous item - re-navigating in the entire video is cumbersome, and this kind of control would make it easier.
3. Improve ability to navigate to exact time point: For videos which are not very short, it is extremely difficult to navigate to an exact position within the file. Using your thumb, you can get to about-maybe-around what you were looking for. To find the exact point you need finer-tune navigation controls.

**Comment by SameenAhnaf** [comment]
If I'm not wrong, this is a duplicate of https://github.com/TeamNewPipe/NewPipe/issues/6271.

**Comment by eyalroz** [comment]
@SameenAhnaf : I don't think so... this is not about gestures, this is about visual controls.

**Comment by SameenAhnaf** [comment]
The screenshot marks the seek bar. That's why, it made me confused. 😅

Note that this was rejected before for main player in https://github.com/TeamNewPipe/NewPipe/issues/3653#issuecomment-909401268.

However, you may use it on background or popup player. When the background or popup player is active, click on notification player.

<img src="https://user-images.githubusercontent.com/79245641/151692257-f492fc46-3c3c-44b4-a567-7427d1b29372.jpg" height="150px"/>

You can turn down seek duration for these buttons to as low as 5 seconds. Go to `Settings`>`Video and Audio`>`Fast Forward/-Rewind Seek Duration`> Select `5 seconds`

**Comment by eyalroz** [comment]
@SameenAhnaf : The issue you linked to does not reject this feature request. It discusses this feature on Android TV, and if I understood correctly, it is explained that the feature is _available_ on Android TV / when using a "DPad" (I don't really know what that is). I'm talking about the player on (Android) mobile phones. I'll clarify that in the opening post.

Can you explain how Fast Fwd / Fast Rewird are supposedly available on mobile phones?

**Comment by SameenAhnaf** [comment]
@eyalroz https://youtu.be/LJev4zA7KL8

**Comment by eyalroz** [comment]
1. I'm asking for a visible control, not a gesture.
2. I watched the video you linked to, couldn't understand how to seek forwards and backwards; so whatever is implement right now - it is such that some (I would say most) users are unable to notice and use.

**Comment by infinitewaveparticle** [comment]
I like the idea of being able to customize this button down to a single frame and be able to do it while paused... So for example, in settings, you pick seconds or frames and then choose the number.

**Comment by SpitFire-666** [comment]
I'd love some visual buttons for seeking. VLC has best of both worlds, you get buttons and can use gestures:

![image](https://user-images.githubusercontent.com/38451588/176085528-df01ec19-f308-4f26-bffa-005f3cd55384.png)

![image](https://user-images.githubusercontent.com/38451588/176085719-47fd0d55-84c9-4b8a-b9d8-2b7642aca7de.png)

VLC's gestures are great for seek control, selecting and dragging the playback cursor in NewPipe is quite fiddly for me, especially since it's right along the bottom in full screen.

Also I just found out about the double tap on the sides to skip 10 seconds, never knew it was there. Thanks!
